python/paddle/fluid/tests/unittests/test_conv3d_transpose_op.py

```diff
@@ -110,7 +110,10 @@ def _get_padding_with_SAME(input_shape, kernel_size, kernel_stride):
 class TestConv3dTransposeOp(OpTest):
     def setUp(self):
         # init as conv transpose
-        self.use_cudnn = False
+        if core.is_compiled_with_cuda():
+            self.use_cudnn = True
+        else:
+            self.use_cudnn = False
```

Comment:
1. If there any other unit testing check the non-cudnn implementation？
2. You can try double for gradient check